src/python_testing/TC_CADMIN_1_19.py

```diff
@@ -0,0 +1,173 @@
+
+        self.step(6)
+        # TH reads the CommissionedFabrics attributes from the Node Operational Credentials cluster
+        current_fabrics = await self.read_single_attribute_check_success(dev_ctrl=self.th1, fabric_filtered=False, endpoint=0, cluster=OC_cluster, attribute=OC_cluster.Attributes.SupportedFabrics)
+        if current_fabrics != max_fabrics:
+            asserts.fail(f"Expected number of fabrics not correct, instead was {current_fabrics}")
+
+        self.step(7)
+        params = await self.openCommissioningWindow(dev_ctrl=self.th1, node_id=self.dut_node_id)
+
+        self.step(8)
+        # TH creates a controller on a new fabric and commissions DUT_CE using that controller
+        fids_ca_dir[current_fabrics + 1] = self.certificate_authority_manager.NewCertificateAuthority()
+        fids_fa_dir[current_fabrics + 1] = fids_ca_dir[current_fabrics +
+                                                       1].NewFabricAdmin(vendorId=0xFFF1, fabricId=current_fabrics + 1)
+        try:
+            fids[current_fabrics + 1] = fids_fa_dir[current_fabrics + 1].NewController(nodeId=current_fabrics + 1)
+            await fids[current_fabrics + 1].CommissionOnNetwork(
+                nodeId=self.dut_node_id, setupPinCode=params.commissioningParameters.setupPinCode,
+                filterType=ChipDeviceCtrl.DiscoveryFilterType.LONG_DISCRIMINATOR, filter=params.randomDiscriminator)
+
+        except ChipStackError as e:
+            # When attempting to create a new controller we are expected to get the following response:
+            # src/credentials/FabricTable.cpp:833: CHIP Error 0x0000000B: No memory
+            # Since the FabricTable is full and unable to create any new fabrics
+            asserts.assert_equal(e.err,  0x0000000B,
+                                 "Expected to return table is full since max number of fabrics has been created already")
+
+        self.step(9)
+        for thc in fids.keys():
+            # Make sure that current test step is 11 (9 + 2 since 5a and 5b test steps included in count), resets here after each loop
+            self.current_step_index = 11
+
+            self.step("9a")
+            fabric_index = await self.read_single_attribute_check_success(dev_ctrl=fids[thc], endpoint=0, cluster=OC_cluster, attribute=OC_cluster.Attributes.CurrentFabricIndex)
+
+            self.step("9b")
+            removeFabricCmd = Clusters.OperationalCredentials.Commands.RemoveFabric(fabric_index)
+            await self.th1.SendCommand(nodeid=self.dut_node_id, endpoint=0, payload=removeFabricCmd)
+
+        self.step(10)
+        # TH reads the CommissionedFabrics attributes from the Node Operational Credentials cluster.
+        current_fabrics = await self.read_single_attribute_check_success(dev_ctrl=self.th1, fabric_filtered=False, endpoint=0, cluster=OC_cluster, attribute=OC_cluster.Attributes.CommissionedFabrics)
+        if current_fabrics != initial_number_of_fabrics:
```

Comment:
Why not just use asserts.assert_equals(current_fabrics, initial_number_of_fabrics)

Reply:
Same comment for all the current if statements that then have an asserts.fail after that

Reply:
This is a good idea! 
I have changed to using this asserts method in place of using the if statements.